### DISCUSSION TITLE
INCLUSION: Return level using Kriging and covariance model from a neural network.

### DISCUSSION THREAD

**Discussion by ibkvictor** [GeoStat-Framework/GSTools#338] [Ideas]
Hi there,

I am working on a neural network for spatial extremes simulation. The model hopes to learn / better estimate dependent or independent extremes. I intend to add some of the implementation as part of the gstools toolset. How do I go about this?

Best,
Victor.

**Comment by LSchueler** [comment]
Hi Victor,

I'm not exactly sure what kind of an answer you are hoping for, as your question is fairly general. As this seems to be a topic which doesn't fit into any of the existing submodules like `field`, `krige`, `variogram`, ..., I would create a new folder in `gstools/src/gstools/` and start there.
Have you thought about maybe creating your own project and using GSTools as a package from the outside? With so little information I cannot really say if your project fits into the scope of GSTools.
We have several other projects, all gathered as the [GeoStat Framework](https://geostat-framework.readthedocs.io/en/latest/).
But who knows, maybe GSTools would be a perfect match.
